js/samples/quickstart-java/src/main/webapp/index.jsp

```diff
@@ -75,7 +75,7 @@
 
             const token = await getImmersiveReaderTokenAsync();
 
-            ImmersiveReader.launchAsync(token, '<%= Microsoft.ImmersiveReader.Constants.SUBDOMAIN %>', data, options)
+            ImmersiveReader.launchAsync(token, '<%= Microsoft.ImmersiveReader.GetAuthTokenServlet.dotenv.get("SUBDOMAIN") %>', data, options)
```

Comment:
can you use GetAuthTokenServlet.SUBDOMAIN instead of this dotenv.get("SUBDOMAIN")